packages/rpc-methods/src/restricted/manageState.test.ts

```diff
@@ -1,18 +1,47 @@
+import { encrypt } from '@metamask/browser-passworder';
+import {
+  MOCK_LOCAL_SNAP_ID,
+  MOCK_SNAP_ID,
+  TEST_SECRET_RECOVERY_PHRASE,
+} from '@metamask/snaps-utils/test-utils';
+import { ethErrors } from 'eth-rpc-errors';
+
 import {
   getManageStateImplementation,
   getValidatedParams,
   ManageStateOperation,
   specificationBuilder,
 } from './manageState';
 
+Object.defineProperty(global, 'crypto', {
+  value: {
+    /* eslint-disable @typescript-eslint/no-require-imports, @typescript-eslint/no-var-requires */
+    ...require('crypto').webcrypto,
+    subtle: require('crypto').webcrypto.subtle,
+    /* eslint-enable @typescript-eslint/no-require-imports, @typescript-eslint/no-var-requires */
+    getRandomValues: (input: Uint8Array) => input.fill(0),
+  },
+});
+
+// Encryption key for `MOCK_SNAP_ID`.
+const ENCRYPTION_KEY =
```

Comment:
Could we use `deriveEntropy` for these just for sanity?

Reply:
It would complicate things, as we can't use top-level await. These values are unlikely to change in the future, so this seems cleaner to do.

Reply:
🤔